src/main/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -368,6 +368,23 @@ public long estimateKeys() {
         return client.getListStrValue("column_family/sstables/by_key/" + getCFName(), queryParams);
     }
 
+    /**
+     * Returns a list of filenames that contain the given key on this node
+     * @param key
+     * @param hexFormat if key is in hex string format
+     * @return list of filenames containing the key
+     */
+    @Override
+    public List<String> getSSTablesForKey(String key, boolean hexFormat)
+    {
+        log(" getSSTablesForKey(String key)");
+        MultivaluedMap<String, String> queryParams = new MultivaluedHashMap<String, String>();
+        queryParams.add("key", key);
+        if (hexFormat) {
+            queryParams.add("hf", "True");
+        }
```

Comment:
hf? can we have a more descriptive name?

Like "format=text" or "format=hex"

Reply:
@avikivity sure, just forced push a new version